src/adaptors/star_citizen.py

```diff
@@ -38,9 +38,11 @@ def __validate_file(self, data):
             raise Exception("File is not a valid Star Citizen XML")
         else:
             if (
-                len(parsed_xml.getElementsByTagName("devices")) == 1
```

Comment:
Is devices no longer a valid construct in latest version?

Reply:
JD 1.4 work with the exported format of the current SC 3.13 release.
However, SC 3.13 now dump an xml automatically, but this one doesn't have the device class.

There are other minors change but it doesn't affect the parser.